requirements.txt

```diff
@@ -0,0 +1,38 @@
+# for creating beautiful command line interfaces
```

Comment:
You do not need to add comments to the requirements .txt file. Just pip freeze your environment and store it in the requirements.txt. Do not go back to edit it.